### PR TITLE
spec for session controller

### PR DESCRIPTION
**Pull request by ductvh-3178**
## Related Tickets
-51120

## WHAT (optional)
- Write rspec for sessions controller,
- 100% code coverage

## HOW
- test case 1: login with not exits email
- test case 2: login with exist email and correct password
- test case 3: login with exist email and incorrect password
- test case 4: login with exist email, correct password and remember params
- test case 5: with exist email and incorrect password
- test case 6: logout

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: spec/controllers/sessions_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+include SessionsHelper
+RSpec.describe SessionsController, type: :controller do
+
+  let!(:user) {FactoryBot.create :user, email:"user@gmail.com", password: "123123"}
+  describe "POST /create" do
+    context "not exist email" do
```
Comment (duypn-2245) [inline spec/controllers/sessions_controller_spec.rb]:
```suggestion
    context " when email not exist" do
```

File: spec/controllers/sessions_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      before do
+        post :create, params: {session: {email: "21354abs@"}}
+      end
+
+      it "login with not exist email" do
```
Comment (duypn-2245) [inline spec/controllers/sessions_controller_spec.rb]:
```suggestion
      it "should not login with not exist email" do
```

Comment (duypn-2245) [reply spec/controllers/sessions_controller_spec.rb]: update câu từ lại đi e

File: spec/controllers/sessions_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        expect(flash[:danger]).to eq I18n.t("sessions.create.user_not_found")
+      end
+    end
+  end
+
+  context "exist email and correct password" do
+    before do
+      post :create, params: {session: {email: "user@gmail.com", password: "123123"}}
+    end
+    it "login in" do
```
Comment (duypn-2245) [inline spec/controllers/sessions_controller_spec.rb]:
```suggestion
    it "should login in success" do
```

**Comment by ductvh-3178** [comment]
ready

**Pull request merged**